main.py

```diff
@@ -70,7 +70,8 @@ def handle_args():
         help="The host address for the MQTT server. Expected to be a string.",
     )
     parser.add_argument(
-        "--mqtt_topic", help="The MQTT topic to subscribe to. Expected to be a string."
+        "--mqtt_command_topic",
```

Comment:
I wonder if it is a good idea to encapsulate all this configuration behind a dedicated class instead of having this in main. What do you think?

Reply:
The code that defines the command-line args is only ever going to be applicable to main.py, so there's no reusability value in having it in a separate class - but it does make it testable and would include it in coverage checks. Same for the code that combines the command-line args with the config settings. So it sounds like a hassle but probably a good idea :)

Reply:
Yeah I get it. No strong feelings either way